plugins/wasi_http/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2019-2024 Second State INC
+
+include(FetchContent)
+FetchContent_Declare(cpr GIT_REPOSITORY https://github.com/libcpr/cpr.git
```

Comment:
Because we are introducing a new dependency here, we should add it to the `licenses.spdx` file.